matlab_octave/exercise_01.m

```diff
@@ -61,9 +63,9 @@
 mules_client.flushdata();  % Flush old data from MuLES       
 tone(500,500); % Beep sound
              
-disp(' Press ESC in the raw EEG signal figure window to break the While Loop');
+disp(' Close the raw EEG signal figure window to break the While Loop');
```

Comment:
I think that the reference to the while loop might be obscure for someone who doesn't code much. This could be something like this instead: "Close the raw EEG signal figure to terminate the example."